File: cont/base/springcontent/shaders/GLSL/Particles/MissileParticleGenerator.lua
return {
	MainCode =
[[
	vec4 lightYellow = GetColorFromIntegers(uvec4(255, 210, 180, 1));

	AddEffectsQuadCamera(
		drawOrder,
		vec3(1.0),
		pos, vec2(fsize), texCoord,
		lightYellow
	);
]]
}

File: rts/Rendering/Env/Particles/Generators/MissileParticleGenerator.cpp
#include "MissileParticleGenerator.h"

CR_BIND(MissileParticleData, )
CR_REG_METADATA(MissileParticleData,
(
	CR_MEMBER(pos),
	CR_MEMBER(fsize),
	CR_MEMBER(speed),
	CR_MEMBER(drawOrder),
	CR_MEMBER(texCoord)
))